components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -96,13 +96,12 @@ double GetPopRecency(const mojom::FeedItemMetadataPtr& article) {
 
   auto& publish_time = article->publish_time;
 
-  double popularity = article->pop_score == 0
-                          ? features::kBraveNewsPopScoreFallback.Get()
-                          : article->pop_score;
+  double popularity = article->pop_score;
   double multiplier = publish_time > base::Time::Now() - base::Hours(5) ? 2 : 1;
   auto dt = base::Time::Now() - publish_time;
 
-  return multiplier * popularity *
+  return (multiplier * popularity +
```

Comment:
So this is removing the recency boost from articles which don't have a popularity - is this what we want?

Reply:
From offline discussion. These were the ideas around the change:

1. given two articles with the same recency (second part of the equation is the same), we should generally rank higher articles that do have a popularity signal.
2. for articles with different recencies, it’s a matter of striking the balance between popularity and recency, that’s why I’ve added a sum rather than the step function we had before. The issue with the step function is that if an article had low popularity (still a stronger signal than no signal) they would be disadvantaged compared to articles with no popularity.

We might want to increase the default recency score to make the trade-off more balanced. As you also brought up, we should address also the case of direct feed and how to add them in the feed, but, if you're okay with it, we could address this issue in a separate PR.